examples/demo-rollup/benches/rollup_coarse_measure.rs

```diff
@@ -0,0 +1,222 @@
+use anyhow::Context;
+use demo_stf::app::NativeAppRunner;
+use demo_stf::runner_config::from_toml_path;
+use std::env;
+use std::sync::Arc;
+
+use sov_demo_rollup::config::RollupConfig;
+use sov_demo_rollup::rng_xfers::RngDaService;
+
+use risc0_adapter::host::Risc0Verifier;
+use sov_db::ledger_db::{LedgerDB, SlotCommit};
+use sov_rollup_interface::mocks::{TestBlock, TestBlockHeader, TestHash};
+use sov_rollup_interface::services::stf_runner::StateTransitionRunner;
+
+use const_rollup_config::SEQUENCER_DA_ADDRESS;
+use demo_stf::genesis_config::create_demo_genesis_config;
+use sov_modules_api::default_signature::private_key::DefaultPrivateKey;
+use sov_rollup_interface::services::da::DaService;
+use sov_rollup_interface::stf::StateTransitionFunction;
+use std::fs;
+use std::io;
+use std::time::{Duration, Instant};
+
+use prometheus::Histogram;
+use prometheus::HistogramOpts;
+use prometheus::Registry;
+
+#[macro_use]
+extern crate prettytable;
+use prettytable::Table;
+
+fn remove_dir_if_exists<P: AsRef<std::path::Path>>(path: P) -> io::Result<()> {
+    if path.as_ref().exists() {
```

Comment:
Let's make `test_helper` and put `remove_dir_if_exists` there it is implemented in two places.